modules/yardstick/src/main/java/org/apache/ignite/yardstick/cache/load/IgniteCacheRandomOperationBenchmark.java

```diff
@@ -390,35 +470,35 @@ private Object createRandomValue(int id, String cacheName) {
      * @return Random value class.
      */
     private Class randomValueClass(String cacheName) {
-        Class[] values;
 
-        if (valuesCacheClasses.containsKey(cacheName))
-            values = valuesCacheClasses.get(cacheName);
-        else
-            values = ModelUtil.valueClasses();
+        Class[] values = valuesCacheClasses.containsKey(cacheName)
+            ? valuesCacheClasses.get(cacheName) : ModelUtil.valueClasses();
 
         return values[nextRandom(values.length)];
     }
 
     /**
+     * @param map Parameters map.
      * @param withoutTransactionCache Without transaction cache.
      * @throws Exception If fail.
      */
-    private void executeOutOfTx(boolean withoutTransactionCache) throws Exception {
+    private void executeOutOfTx(Map<Object, Object> map, boolean withoutTransactionCache) throws Exception {
         for (IgniteCache cache : availableCaches) {
```

Comment:
"map" is defined as a member class variable. I don't see any reason why you should pass it to methods as a parameter.